src/components/user/ProposalsPeopleTable.tsx

```diff
@@ -0,0 +1,514 @@
+              type="email"
+              component={TextField}
+              margin="normal"
+              fullWidth
+              flex="1"
+              data-cy="email"
+            />
+            <Button
+              data-cy="findUser"
+              variant="contained"
+              color="primary"
+              type="submit"
+              className={classes.inviteButton}
+            >
+              Find User
+            </Button>
+          </Form>
+        </div>
+      </>
+    );
+  };
+
+  return (
+    <Formik
+      initialValues={{
+        email: '',
+      }}
+      initialErrors={{
+        email: 'test',
+      }}
+      onSubmit={async (values, { setFieldError, setFieldValue }) => {
+        // If there is an email and it has not already been searched
+        if (values.email && !tableEmails.includes(values.email)) {
+          setLoading(true);
+          const userDetails = await getUserByEmail(values.email, sendRequest);
+
+          if (!userDetails) {
+            setDisplayError(true);
+            setFieldError('email', 'Please enter valid email address');
+            setLoading(false);
+
+            return;
+          }
+
+          if (props.selectedUsers?.includes(userDetails.id)) {
+            setFieldError('email', 'User is already on the proposal');
+            setLoading(false);
+
+            return;
+          }
+
+          if (!invitedUsers.some((user) => user.id === userDetails.id)) {
+            //Add users to the table
+            setInvitedUsers([userDetails].concat(invitedUsers));
+            setTableEmails(tableEmails.concat([values.email]));
+            setFieldValue('email', '');
+
+            //If we are selecting multiple users add the user as pre selected.
+            if (props.selection)
+              setSelectedParticipants(
+                selectedParticipants.concat([userDetails])
+              );
+
+            //This is the recommend way to refresh a table but it is unsupported in typescript and may not be supported anytime soon
+            //See: Refresh Data Example https://material-table.com/#/docs/features/remote-data
+            //and see: https://github.com/mbrn/material-table/issues/1752
+            // @ts-ignore
+            tableRef.current.onQueryChange();
+          } else {
+            setFieldError('email', 'Could not add user to Proposal');
+          }
+          setLoading(false);
+        } else if (tableEmails.includes(values.email)) {
+          setFieldError(
+            'email',
+            'User has already been added, possibly use filter instead'
+          );
+        } else {
+          setFieldError('email', 'Please enter a email');
+        }
+      }}
+    >
+      {() => (
+        <div data-cy="co-proposers" className={classes.tableWrapper}>
+          <MaterialTable
+            tableRef={tableRef}
+            icons={tableIcons}
+            title={
+              <Typography variant="h6" component="h2">
```

Comment:
Is this the "Add Co-Proposers" title?

If so, I think the header component might want to be `h1` since it is the first heading of the modal (we decided as part of [this issue](https://github.com/UserOfficeProject/stfc-user-office-project/issues/117) to treat modals as the start of a new heading hierarchy.)
```suggestion
              <Typography variant="h6" component="h1">
```

Reply:
Sorry forgot to reply earlier, thanks for catching this!